src/Microsoft.FeatureManagement/FeatureManager.cs

```diff
@@ -211,6 +289,79 @@ private IFeatureFilterMetadata GetFeatureFilterMetadata(string filterName)
             return filter;
         }
 
+        private IFeatureVariantAssignerMetadata GetFeatureAssignerMetadata(string assignerName)
+        {
+            IFeatureVariantAssignerMetadata assigner = _assignerMetadataCache.GetOrAdd(
+                assignerName,
+                (_) => {
+
+                    IEnumerable<IFeatureVariantAssignerMetadata> matchingAssigners = _variantAssigners.Where(f =>
+                    {
+                        string name = GetMetadataName(f.GetType());
+
+                        return IsMatchingMetadataName(name, assignerName);
```

Comment:
>IsMatchingMetadataName [](http://example.com/codeflow?start=31&length=22)

Is the filter name matching logic same as the assigner name matching too? `IsMatchingMetadataName` has some namespacing logic too. Does that apply to assigners as well?

Reply:
Yes it does.